src/sage/schemes/elliptic_curves/kodaira_symbol.py

```diff
@@ -333,11 +407,24 @@ def KodairaSymbol(symbol):
         [I0*, II*, III*, IV*, I1*, I2*, I3*, I4*, I5*]
         sage: all(KS(str(KS(n))) == KS(n) for n in range(-10,10) if n != 0)
         True
+        sage: unicode_art(KS(2))
+        <BLANKLINE>
+         \     /
+          \   /
+           \ /
+            |
+         1  |
+        <BLANKLINE>
+        <BLANKLINE>
+
```

Comment:
remove the empty lines at the end of doctests ; try also without the line with only <BLANKLINE> (it should work, I think)